crates/cxx-qt-macro/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+# SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+# SPDX-FileContributor: Gerhard de Clercq <gerhard.declercq@kdab.com>
+# SPDX-FileContributor: Leon Matthes <leon.matthes@kdab.com>
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0
+[package]
+name = "cxx-qt-macro"
+version.workspace = true
+authors = ["Andrew Hayzen <andrew.hayzen@kdab.com>","Leon Matthes <leon.matthes@kdab.com>", "Gerhard de Clercq <gerhard.declercq@kdab.com>"]
+edition.workspace = true
+license.workspace = true
+description = "A macro for Qt/C++ interop in Rust"
+homepage = "https://kdab.github.io/cxx-qt/book/"
+readme = "README.md"
+keywords = ["cxx", "ffi", "QML", "Qt"]
```

Comment:
do our other sub crates we haven't been putting keywords, categories, homepage on them. Wonder if we should or shouldn't and just leave them for the main crate or have them everywhere? :thinking:

Reply:
Removed it